packages/wrangler/e2e/wrangler-mixed-mode-resources.test.ts

```diff
@@ -0,0 +1,309 @@
+				`wrangler d1 execute --remote ${name} --file schema.sql`
+			);
+			return { id, name };
+		},
+		generateWranglerConfig: ({ id, name }) => ({
+			name: "mixed-mode-d1-test",
+			main: "d1.js",
+			compatibility_date: "2025-01-01",
+			d1_databases: [
+				{
+					binding: "DB",
+					database_id: id,
+					database_name: name,
+					remote: true,
+				},
+			],
+		}),
+		expectedResponseMatch: "existing-value",
+	},
+];
+
+describe("Wrangler Mixed Mode E2E Tests", () => {
+	describe.each(testCases)("$name", (testCase) => {
+		let helper: WranglerE2ETestHelper;
+
+		beforeEach(() => {
+			helper = new WranglerE2ETestHelper();
+		});
+
+		it("works with mixed mode enabled", async () => {
+			await helper.seed(
+				path.resolve(__dirname, "./seed-files/mixed-mode-workers")
+			);
+
+			const setupResult = (await testCase.setup?.(helper)) ?? {};
+
+			const wranglerConfig = testCase.generateWranglerConfig(setupResult);
+			await helper.seed({
+				"wrangler.json": JSON.stringify(wranglerConfig, null, 2),
+			});
+
+			const worker = helper.runLongLived("wrangler dev --x-mixed-mode");
+
+			const { url } = await worker.waitForReady();
+
+			const response = await fetchText(url);
+			expect(response).toMatch(testCase.expectedResponseMatch);
+		});
+
+		it.skipIf(testCase.worksWithoutMixedMode)(
+			"fails when mixed mode is disabled",
+			async () => {
+				await helper.seed(
+					path.resolve(__dirname, "./seed-files/mixed-mode-workers")
+				);
+
+				const setupResult = (await testCase.setup?.(helper)) ?? {};
+
+				const wranglerConfig = testCase.generateWranglerConfig(setupResult);
+				await helper.seed({
+					"wrangler.json": JSON.stringify(wranglerConfig, null, 2),
+				});
+
+				const worker = helper.runLongLived("wrangler dev");
+
+				const { url } = await worker.waitForReady();
+
+				const response = await fetchText(url);
+				expect(response).not.toMatch(testCase.expectedResponseMatch);
+			}
+		);
+	});
+
+	describe.sequential("Sequential mixed mode tests with worker reloads", () => {
+		let worker: WranglerLongLivedCommand;
+		let helper: WranglerE2ETestHelper;
+
+		let url: string;
+
+		beforeAll(async () => {
+			helper = new WranglerE2ETestHelper();
+			await helper.seed(
+				path.resolve(__dirname, "./seed-files/mixed-mode-workers")
+			);
+
+			await helper.seed({
+				"wrangler.json": JSON.stringify(
+					{
+						name: "mixed-mode-sequential-test",
+						main: "placeholder.js",
+						compatibility_date: "2025-01-01",
+					},
+					null,
+					2
+				),
+				"placeholder.js":
+					"export default { fetch() { return new Response('Ready to start tests') } }",
+			});
+
+			worker = helper.runLongLived("wrangler dev --x-mixed-mode", {
+				cleanup: false,
+			});
+
+			const ready = await worker.waitForReady();
+			url = ready.url;
+		});
+		afterAll(async () => {
+			await worker.stop();
+		});
+
+		it.each(testCases)("$name", async (testCase) => {
+			console.log(`Testing ${testCase.name} with worker reload...`);
```

Comment:
Move this console message to the title of the it clause?

```ts
it.each(testCases)("$name with worker reload...`", async (testCase) => {
```

Reply:
https://github.com/cloudflare/workers-sdk/pull/9327/commits/81b8737bdac9a0016e9d91b4e71a011755372b95